docs/user/using-the-capture-app.md

```diff
@@ -308,6 +309,14 @@ To see a rule being executed while enrolling a tracked entity instance you will
 5. You will now be able to see the warning produced by the program rule underneath the birth date field. 
 ![](resources/images/program-rules-on-creation-02.png)
 
+### Leaving the page
+
+If the user attempts to change their context, either by changing the program or the organisation unit from the locked selector or by clicking the **Clear selections** button the outcome will depend on the page state:
+1. The user has not typed anything in the fields: The context is updated without any message and the user is redirected.
+2. The user has already pressed the **Save** button, and a new registration has been sent to the server, the redirection to enrollment dashboard page will be cancelled and user will be navigated to the new context page. 
+3. The user has entered details into the fields, see a modal warning them that they will lose their data.
+
+
```

Comment:
This doesn't seem relevant to the user documentation, especially referring to `the user`. This is a technical description of the different departure flows possible from this page, but I don't think they need to be documented in user documentation. The user is informed from the interface.

(Aside: user documentation should not refer to "the user". That term is meant for technical design documentation.)